src/platform/site-wide/index.js

```diff
@@ -6,6 +6,7 @@ import '../monitoring/sentry.js';
 import './legacy/menu'; // Used in the footer.
 import './moment-setup';
 import './popups';
+import '../../applications/static-pages/alerts-dismiss-view';
```

Comment:
Can we move this import to the [static-pages-entry](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/static-pages/static-pages-entry.js#L21)?

Reply:
Sure thing, @ncksllvn - done.